Cargo.toml

```diff
@@ -15,4 +15,5 @@ rustc-serialize = "*"
 cbor = "*"
 sodiumoxide = "*"
 rand = "*"
-routing = "0.1.1"
+[dependencies.routing]
+git = "https://github.com/dirvine/routing.git"
```

Comment:
I don't think we should be reverting back to github - Fraser is working on a PR for Routing that when merged - a new crate will be published and the version can be updated then